Vic2ToHoI4/Source/V2World/Pop.cpp

```diff
@@ -62,19 +33,17 @@ Vic2::Pop::Pop(const std::string& typeString, std::istream& theStream):
 			auto equalsSign = getNextTokenWithoutMatching(theStream);
 			auto issueSupport = getNextTokenWithoutMatching(theStream);
 
-			std::string issueName = issuesInstance.getIssueName(std::stoi(*possibleIssue));
+			std::string issueName = theIssues.getIssueName(std::stoi(*possibleIssue));
 			popIssues.insert(std::make_pair(issueName, std::stof(*issueSupport)));
 
 			possibleIssue = getNextTokenWithoutMatching(theStream);
 		}
 	});
-	registerKeyword(std::regex("id"), [this](const std::string& unused, std::istream& theStream)
-	{
+	registerKeyword(std::regex("id"), [this](const std::string& unused, std::istream& theStream) {
 		commonItems::singleInt idInt(theStream);
 		id = idInt.getInt();
 	});
-	registerKeyword(std::regex("[a-z\\_]+"), [this](const std::string& cultureString, std::istream& theStream)
-	{
+	registerKeyword(std::regex("[a-z\\_]+"), [this](const std::string& cultureString, std::istream& theStream) {
```

Comment:
A-Z0-9. They may not exist in vanilla vic2, but may be imported from modded vic2.